dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/intermediate/components/Iso2CountryCodesMultiSelectComponent.kt

```diff
@@ -0,0 +1,108 @@
+package org.dataland.frameworktoolbox.intermediate.components
+
+import org.apache.commons.text.StringEscapeUtils.escapeEcmaScript
+import org.dataland.frameworktoolbox.intermediate.FieldNodeParent
+import org.dataland.frameworktoolbox.intermediate.components.support.SelectionOption
+import org.dataland.frameworktoolbox.intermediate.datapoints.NoDocumentSupport
+import org.dataland.frameworktoolbox.intermediate.datapoints.addPropertyWithDocumentSupport
+import org.dataland.frameworktoolbox.specific.datamodel.TypeReference
+import org.dataland.frameworktoolbox.specific.datamodel.elements.DataClassBuilder
+import org.dataland.frameworktoolbox.specific.fixturegenerator.elements.FixtureSectionBuilder
+import org.dataland.frameworktoolbox.specific.uploadconfig.elements.UploadCategoryBuilder
+import org.dataland.frameworktoolbox.specific.uploadconfig.functional.FrameworkUploadOptions
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.SectionConfigBuilder
+import org.dataland.frameworktoolbox.specific.viewconfig.elements.getTypescriptFieldAccessor
+import org.dataland.frameworktoolbox.specific.viewconfig.functional.FrameworkDisplayValueLambda
+import org.dataland.frameworktoolbox.utils.capitalizeEn
+import org.dataland.frameworktoolbox.utils.typescript.TypeScriptImport
+import org.dataland.frameworktoolbox.utils.typescript.generateTsCodeForSelectOptionsMappingObject
+
+/**
+ * An ISO2 Country Code represents a selection of string-options generated from country Codes. Multiple entries can be
+ * selected.
+ */
+open class Iso2CountryCodesMultiSelectComponent(
+    identifier: String,
+    parent: FieldNodeParent,
+) : ComponentBase(identifier, parent, "java.util.EnumSet") {
+
+    var options: Set<SelectionOption> = mutableSetOf()
```

Comment:
Wo kommen die options am Ende her?

Reply:
die werden seitens der toolobx nicht benutzt. in der upload page werden sie nachher über getIsoCodes() oder so ähnlich im frontend geparst

Reply:
da ich sie nicht löschen kann, hab ich sie einfach leer gesetzt

Reply:
Got removed due to broken backend